provider/pkg/metadata/annotations.go

```diff
@@ -162,7 +165,10 @@ func GetReadyCondition(
 // SkipReadyCondition returns true if the inputs are annotated such that we
 // should not await readiness.
 func SkipReadyCondition(inputs *unstructured.Unstructured) bool {
-	return IsAnnotationTrue(inputs, AnnotationSkipAwait)
+	if IsAnnotationTrue(inputs, AnnotationSkipAwait) {
+		return true
+	}
+	return strings.EqualFold(GetAnnotationValue(inputs, AnnotationSkipAwait), "ready")
```

Comment:
I would advocate for parsing the value as a comma-separated string, and then checking for the enum values of true, ready, delete.  This would allow for a user to do `ready,delete` and for us to contemplate treating `true` as `ready`.